arch/cpu/cc26xx-cc13xx/rf-core/rf-core.c

```diff
@@ -399,6 +399,9 @@ rf_core_set_modesel()
   } else if(chip_type == CHIP_TYPE_CC1350) {
     HWREG(PRCM_BASE + PRCM_O_RFCMODESEL) = PRCM_RFCMODESEL_CURR_MODE5;
     rv = RF_CORE_CMD_OK;
+  } else if (chip_type == CHIP_TYPE_CC2640R2) {
+	  HWREG(PRCM_BASE + PRCM_O_RFCMODESEL) = PRCM_RFCMODESEL_CURR_MODE1;
+	  rv = RF_CORE_CMD_OK;
```

Comment:
Please fix the indentation here. Use spaces please (as well as anywhere else you may have used tabs)

Reply:
```
  } else if(chip_type == CHIP_TYPE_CC1350) {
    HWREG(PRCM_BASE + PRCM_O_RFCMODESEL) = PRCM_RFCMODESEL_CURR_MODE5;
    rv = RF_CORE_CMD_OK;
#ifdef ThisLibraryIsFor_CC26x0R2_HaltIfViolated
  } else if (chip_type == CHIP_TYPE_CC2640R2) {
	  HWREG(PRCM_BASE + PRCM_O_RFCMODESEL) = PRCM_RFCMODESEL_CURR_MODE1;
	  rv = RF_CORE_CMD_OK;
#endif
  }
```